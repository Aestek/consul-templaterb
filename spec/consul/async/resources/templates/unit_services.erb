Services
<%
  services().each_pair do |service_name, tags|
%><%= service_name %>:<%= tags.map {|tag| tag}.join(' ') %>
<%
  end
%>
Services with filter http
<%
  services(tag: 'http').each_pair do |service_name, tags|
%><%= service_name %>:<%= tags.map {|tag| tag}.join(' ') %>
<%
  end
%>
Services with filter coin
<%
  services(tag: 'coin').each_pair do |service_name, tags|
%><%= service_name %>:<%= tags.map {|tag| tag}.join(' ') %>
<%
  end
%>