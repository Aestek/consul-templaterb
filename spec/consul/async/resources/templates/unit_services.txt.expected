Services
consul:http UI

Services with filter http
consul:http UI

Services with filter coin
